### PR TITLE
[15.0][IMP] account_banking_sepa_direct_debit: Button send mandate per email

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Add button to send mandate per email
- Add field to be able to filter if mandate was sent
- Add new mandate template for email
- Add unit test

TT46064

@Tecnativa

@pedrobaeza

### CODE REVIEW

File: account_banking_sepa_direct_debit/__manifest__.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 # Copyright 2016 Tecnativa - Antonio Espinosa
 # Copyright 2014-2022 Tecnativa - Pedro M. Baeza
+# Copyright 2023 Tecnativa - Carolina Fernandez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Account Banking SEPA Direct Debit",
     "summary": "Create SEPA files for Direct Debit",
-    "version": "15.0.2.1.1",
+    "version": "15.0.2.1.2",
```
Comment (pedrobaeza) [inline account_banking_sepa_direct_debit/__manifest__.py]: Don't change version manually.

File: account_banking_sepa_direct_debit/models/account_banking_mandate.py [hunk 2]
```diff
@@ -45,6 +45,7 @@ class AccountBankingMandate(models.Model):
     )
     unique_mandate_reference = fields.Char(size=35)  # cf ISO 20022
     display_name = fields.Char(compute="_compute_display_name2", store=True)
+    mark_mandate_as_sent = fields.Boolean()
```
Comment (pedrobaeza) [inline account_banking_sepa_direct_debit/models/account_banking_mandate.py]: Put simpler name:

```suggestion
    is_sent = fields.Boolean()
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
Functional review. LGTM

**Comment by pedrobaeza** [comment]
/ocabot merge minor

Please fw-port it to v16

**Pull request merged**